tests/Keystore/StoredKeyTests.cpp

```diff
@@ -307,7 +319,7 @@ TEST(StoredKey, RemoveAccount) {
     EXPECT_EQ(key.accounts.size(), 2);
     key.removeAccount(TWCoinTypeEthereum);
     EXPECT_EQ(key.accounts.size(), 1);
-    EXPECT_EQ(key.accounts[0].coin(), coinTypeBc);
+    EXPECT_EQ(key.accounts[0].coin, coinTypeBc);
 }
```

Comment:
can we add example with smart chain?

Reply:
Or test case that will test new and old keystore for active accounts

Reply:
will add a swift test case

Reply:
done